commands.go

```diff
@@ -93,24 +97,38 @@ func doVersion(_ *flag.FlagSet, _ []string) error {
 	return nil
 }
 
-/* +command configtest - configtest
+/*
+	 +command configtest - configtest
 
-	configtest
+		configtest
 
 do configtest
 */
 func doConfigtest(fs *flag.FlagSet, argv []string) error {
 	conf, err := resolveConfig(fs, argv)
 	if err != nil {
-		return fmt.Errorf("failed to test config: %s", err)
+		return fmt.Errorf("\x1b[31m[CRITICAL] failed to test config: %s \x1b[0m", err)
```

Comment:
mackerel-agent runs on Windows, so we had better either to use https://github.com/mattn/go-colorable or https://github.com/fatih/color, or to disable ANSI escape codes except on `go:build unix`.

Reply:
fixed https://github.com/mackerelio/mackerel-agent/pull/813/commits/56a06425e7c9bb2803ed01b2b00119b2b5d95d3e